cmd/tcpgoon.go

```diff
@@ -63,6 +65,13 @@ func validateRequiredArgs(params *tcpgoonParams, args []string) error {
 		return errors.New("Number of required parameters doesn't match")
 	}
 	params.hostPtr = args[0]
+
+	addrs, err := net.LookupIP(params.hostPtr)
+	if err != nil {
+		return errors.New("Domain name not resolvable")
+	}
+	params.hostPtr = addrs[0].String()
```

Comment:
I don't think rewriting the original variable is a good idea. Probably it makes sense to store both, for reporting purposes. Please, consider:
* to inform, at least, in the debug output of the IP address you pick to run the tests against
* what do you think makes more sense: to print in the actual report the IP address we are using, the dns or both (something like `dns(ip)`)?

Reply:
done